apps/opik-backend/src/main/java/com/comet/opik/infrastructure/http/CORS.java

```diff
@@ -0,0 +1,56 @@
+package com.comet.opik.infrastructure.http;
+
+import com.comet.opik.infrastructure.OpikConfiguration;
+import io.dropwizard.core.setup.Environment;
+import jakarta.servlet.DispatcherType;
+import jakarta.servlet.FilterRegistration;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+
+import java.util.EnumSet;
+
+@Slf4j
+public class CORS {
+    private static final String CORS_PATH_FILTER = "/*";
+
+    private static final String[] ALLOWED_HEADERS = new String[]{
+            "Authorization",
+            "X-Requested-With",
+            "Content-Type",
+            "Accept",
+            "Origin",
+            "Comet-Sdk-Api",
+            "comet-username",
+            "comet-react-ver",
+    };
+
+    private static final String[] ALLOWED_METHODS = new String[]{
```

Comment:
Minor: same, there's a constant for this already.

Reply:
Done in https://github.com/comet-ml/opik/pull/498/commits/73e280a2497cf86ddd6656a134705b73e7681ff1